src/data/equiv/ring.lean

```diff
@@ -107,8 +107,12 @@ instance : ring_equiv_class (R ≃+* S) R S :=
 
 instance : has_coe_to_fun (R ≃+* S) (λ _, R → S) := ⟨ring_equiv.to_fun⟩
 
+@[simp] lemma to_equiv_eq_coe (f : R ≃+* S) : f.to_equiv = f := rfl
+
 @[simp] lemma to_fun_eq_coe (f : R ≃+* S) : f.to_fun = f := rfl
 
+@[simp] lemma coe_coe_eq_coe (f : R ≃+* S) : ((f : R ≃ S) : R → S) = f := rfl
```

Comment:
There are sevaral ways to coerce a `ring_equiv`, for example there is also the coercion to `ring_hom`.
```suggestion
@[simp] lemma coe_to_equiv (f : R ≃+* S) : ((f : R ≃ S) : R → S) = f := rfl
```